modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/introspection/ServerInfo.java

```diff
@@ -536,7 +537,15 @@ public static ServerInfo build(String name, String version) {
             } else {
                 bi = server.bundles.get(bundle.getSymbolicName());
             }
-            bi.setDeploymentOrder(bindex++);
+            List<ComponentInfo> components = bi.getComponents();
+            OptionalLong min = components.stream().mapToLong(ComponentInfo::getRegistrationOrder).min();
+            if (min.isPresent()) {
+                bi.setMinRegistrationOrder(min.getAsLong());
+            }
```

Comment:
Could be `components.stream().mapToLong(ComponentInfo::getRegistrationOrder).min().ifPresent(min -> bi.setMinRegistrationOrder(min))`